src/dist/replication/lib/duplication/test/load_from_private_log_test.cpp

```diff
@@ -28,6 +28,24 @@ class load_from_private_log_test : public duplication_test_base
         duplicator = create_test_duplicator();
     }
 
+    // return number of entries written
+    int generate_multiple_log_files(int files_num = 3)
+    {
+        // decree ranges from [1, files_num*10)
```

Comment:
the annotation is wrong when files_num <= 0.
And I suggest use `uint32_t files_num` is better.